core/src/SplitView.html

```diff
@@ -109,6 +113,10 @@ <h1 class="fd-splitView__title">{splitViewSettings.title}</h1>
               splitViewIframe = obj.splitViewIframe;
             } else if (prop === 'splitViewIframeData') {
               splitViewIframeData = obj.splitViewIframeData;
+            } else if (prop === 'splitViewWC') {
+              splitViewWC = obj.splitViewWC;
+            } else if (prop === 'splitViewWCData') {
+              splitViewWC = obj.splitViewWC;
```

Comment:
Should it be `splitViewWCData` or is `splitViewWC` correct?

Reply:
yes `splitViewWCData` is correct, nice catch 👍 , I'll update it